### PR TITLE
Adding a script for releasing FluentUI packages a la fluent-ui-react repo

### PR DESCRIPTION
**Pull request by kenotron**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

This PR adds the new `yarn release:minor` and `yarn release:patch` commands from `fluent-ui-react` repo. The implementation details will change quite soon after this change to allow beachball to take over the publish process. But for now, we'll go with the MVP:

1. adds a few task functions for prepublish (which will "fix up" lerna.json, package.json to limit the scope to the packages inside /packages/fluentui -- the `lerna publish` command doesn't allow `--scope`
2. adds an azure pipeline yml file - this is the expected workflow from now on (after the merge) to make @fluentui/react releases from the repo. We'll queue builds and at the build time ask whether the bump is either `minor` or `patch`

#### Focus areas to test

I've tested this against a local verdaccio instance by running:

```
yarn release:minor --registry http://localhost:4873 --no-push
```

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11950)

### CODE REVIEW

File: azure-pipelines.release-fluentui.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+pr: none
+trigger: none
+
+# Customize build number to include major version
```
Comment (ecraig12345) [inline azure-pipelines.release-fluentui.yml]: comment is wrong?

File: azure-pipelines.release-fluentui.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/OfficeDev/office-ui-fabric-react.git
+    displayName: Authenticate git for pushes
+
+  - script: |
+      yarn
+    displayName: yarn
+
+  - script: |
+      PRODUCTION=true yarn build
+       && yarn lint
```
Comment (ecraig12345) [inline azure-pipelines.release-fluentui.yml]: Does this work on two separate lines?

File: lerna.json [hunk 3]
```diff
@@ -1,11 +1,6 @@
...
-    "packages/*",
-    "apps/*",
-    "scripts",
-    "packages/fluentui/*"
-  ],
+  "packages": ["apps/*", "packages/*", "scripts", "packages/fluentui/*"],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "0.0.0"
+  "version": "0.44.0"
```
Comment (ecraig12345) [inline lerna.json]: Wish there was a way to comment on this file about why we're using this version

File: azure-pipelines.release-fluentui.yml (hunk 2 above)
Comment (kenotron) [reply azure-pipelines.release-fluentui.yml]: oops - a goober

File: lerna.json (hunk 3 above)
Comment (kenotron) [reply lerna.json]: i hate json file - wish this was a .js file

Comment (JasonGore) [reply lerna.json]: :'(

**Pull request merged**